x-pack/plugins/monitoring/public/lib/elasticsearch_settings/checkers/settings_checker.js

```diff
@@ -44,7 +44,10 @@ export class SettingsChecker {
 
   async executeCheck() {
     try {
-      const { data } = await this.$http.get(this.getApi());
+      console.log('EXECUTE CHECK');
```

Comment:
Do we want to keep this `console.log()`?

Reply:
Nope, good catch, I'll remove it 🤦